dev-util/android-studio/android-studio-2024.2.1.10-r1.ebuild

```diff
@@ -0,0 +1,142 @@
+	opt/${PN}/plugins/android/resources/installer/*/*
+	opt/${PN}/plugins/android/resources/native/*
+	opt/${PN}/plugins/android/resources/perfetto/*/*
+	opt/${PN}/plugins/android/resources/screen-sharing-agent/*/*
+	opt/${PN}/plugins/android/resources/simpleperf/*/*
+	opt/${PN}/plugins/android/resources/trace_processor_daemon/*
+	opt/${PN}/plugins/android/resources/transport/*/*
+	opt/${PN}/plugins/android/resources/transport/native/agent/*/*
+	opt/${PN}/plugins/android-ndk/resources/lldb/android/*/*
+	opt/${PN}/plugins/android-ndk/resources/lldb/bin/*
+	opt/${PN}/plugins/android-ndk/resources/lldb/lib/python3.9/lib-dynload/*
+	opt/${PN}/plugins/android-ndk/resources/lldb/lib64/*
+	opt/${PN}/plugins/design-tools/resources/layoutlib/data/linux/lib64/*
+	opt/${PN}/plugins/c-clangd/bin/clang/linux/*/*
+	opt/${PN}/plugins/webp/lib/libwebp/linux/*
+"
+
+DESCRIPTION="Android development environment based on IntelliJ IDEA"
+HOMEPAGE="https://developer.android.com/studio"
+SRC_URI="https://redirector.gvt1.com/edgedl/android/studio/ide-zips/${PV}/${P}-linux.tar.gz"
+S=${WORKDIR}/${PN}
+LICENSE="Apache-2.0 android BSD BSD-2 CDDL-1.1 CPL-0.5
+	EPL-1.0 GPL-2 GPL-2+ JDOM IJG LGPL-2.1 MIT
+	MPL-1.1 MPL-2.0 NPL-1.1 OFL-1.1 ZLIB"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="selinux"
+RESTRICT="bindist mirror strip"
+
+RDEPEND="${DEPEND}
+	selinux? ( sec-policy/selinux-android )
+	>=app-arch/bzip2-1.0.8-r4
+	>=dev-libs/expat-2.5.0
+	>=dev-libs/libffi-3.4.4
+	>=media-libs/fontconfig-2.14.0-r1
+	>=media-libs/freetype-2.12.1-r1
+	>=media-libs/libpng-1.6.39
+	>=media-libs/mesa-22.2.5[X(+)]
+	|| ( gnome-extra/zenity kde-apps/kdialog x11-apps/xmessage x11-libs/libnotify )
+	sys-libs/ncurses-compat:5[tinfo]
+	>=sys-libs/zlib-1.2.13-r1
+	>=x11-libs/libX11-1.8.1
+	>=x11-libs/libXau-1.0.11
+	>=x11-libs/libXdamage-1.1.6
+	>=x11-libs/libXdmcp-1.1.4-r2
+	>=x11-libs/libXext-1.3.5
+	>=x11-libs/libXfixes-6.0.0
+	>=x11-libs/libXrender-0.9.11
+	>=x11-libs/libXxf86vm-1.1.5
+	>=x11-libs/libdrm-2.4.114
+	>=x11-libs/libxcb-1.15-r1
+	>=x11-libs/libxshmfence-1.3.2
+	virtual/libcrypt:=
+"
+
+src_prepare() {
+
+	default
+
+	rm "${S}"/plugins/android/resources/installer/{arm64-v8a,armeabi-v7a,x86}/*
+	rm "${S}"/plugins/android/resources/perfetto/{arm64-v8a,armeabi-v7a,x86}/*
+	rm "${S}"/plugins/android/resources/process-tracker-agent/native/{arm64-v8a,armeabi-v7a,x86}/*
+	rm "${S}"/plugins/android/resources/screen-sharing-agent/{arm64-v8a,armeabi-v7a,x86}/*
+	rm "${S}"/plugins/android/resources/transport/{arm64-v8a,armeabi-v7a,x86}/*
+	rm "${S}"/plugins/android/resources/transport/native/agent/{arm64-v8a,armeabi-v7a,x86}/*
+
+	sed -i \
+		-e "\$a\\\\" \
+		-e "\$a#-----------------------------------------------------------------------" \
+		-e "\$a# Disable automatic updates as these are handled through Gentoo's" \
+		-e "\$a# package manager." \
+		-e "\$a#-----------------------------------------------------------------------" \
+		-e "\$aide.no.platform.update=Gentoo"  bin/idea.properties
```

Comment:
Missing `|| die`. That said, can't you simply

```bash
cat <<-EOF >> bin/idea.properties || die
#-----------------------------------------------------------------------
# Disable automatic updates as these are handled through Gentoo's
# package manager.
#-----------------------------------------------------------------------
aide.no.platform.update=Gentoo
EOF

Reply:
Good catch, fixed!